src/Castle.Core/Castle.Core.csproj

```diff
@@ -3,7 +3,7 @@
 	<Import Project="..\..\buildscripts\common.props"></Import>
 
 	<PropertyGroup>
-		<TargetFrameworks>net40;net45;netstandard1.3;netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>net40;net45;netstandard1.3;netstandard1.5;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
As above, please avoid these whitespace-only changes. This code file (like all others) uses tabs, not spaces.